src/main/java/org/junit/Assert.java

```diff
@@ -812,6 +812,41 @@ public static void assertNotSame(Object unexpected, Object actual) {
         assertNotSame(null, unexpected, actual);
     }
 
+    /**
+     * Asserts that two {@link CharSequence} instances have the same contents.
+     * If they do not, an {@link AssertionError} is thrown.
+     *
+     * @param expected the expected contents
+     * @param actual the object to compare to
+     */
+    public static void assertContentEquals(String expected, CharSequence actual)  {
```

Comment:
Suggestion - why not make the `expected` a `CharSequence` too?

Reply:
See the original discussion: https://github.com/junit-team/junit4/issues/1432#issuecomment-287558614

Reply:
... and also here: https://github.com/junit-team/junit4/pull/1438#discussion_r111419678